corehq/util/datadog/metrics.py

```diff
@@ -1,5 +1,5 @@
 JSERROR_COUNT = 'jserror.count'
 ERROR_COUNT = 'error.count'
-REPEATER_ERROR_COUNT = 'repeater.error.count'
+REPEATER_ERROR_COUNT = 'commcare.repeater.error.count'
```

Comment:
@proteusvacuum It looks like there's a metric called `commcare.repeaters.overdue` could we change this to be `commcare.repeaters.error_count` or something similar so they can have the same prefix?